setup.cfg

```diff
@@ -51,5 +51,8 @@ non_interactive     = True
 [mypy-openfisca_core.commons.tests.*]
 ignore_errors       = True
 
+[mypy-openfisca_core.periods.tests.*]
+ignore_errors 		= True
```

Comment:
What do we get from both adding this file to the list of tests and ignoring the resulting errors? 🤔

Reply:
This line in particular is to ignore enforcing type checks in tests. While I do believe as we've discussed before type hints are great code documentation, I also think there's a time for them: maturity. Proper testing is not this library's best feature, thus the line above. This hypothesis can of course be challenged.

Reply:
That sounds legitimate 🙂 I would welcome a comment on it though.